lib/src/airship_events.dart

```diff
@@ -54,10 +54,9 @@ class MessageCenterUpdatedEvent {
       this.messageUnreadCount, this.messageCount);
 
   static MessageCenterUpdatedEvent fromJson(dynamic json) {
-    var messageUnreadCount = json["messageUnreadCount"];
-    var messageCount = json["messageCount"];
-    return MessageCenterUpdatedEvent._internal(
-        messageUnreadCount, messageCount);
+    final num unread = json["messageUnreadCount"] ?? 0;
+    final num count = json["messageCount"] ?? 0;
+    return MessageCenterUpdatedEvent._internal(unread.toInt(), count.toInt());
```

Comment:
why was this needed?

Reply:
it was also throwing a runtime exception when it was null